docs/getting-started.md

```diff
@@ -37,7 +37,7 @@ to [the AKS installation guide](aks-installation.md).
 ## Importing Cloud VMs
 
 To manage security policies of VMs, we need to first import target VMs onto the
-`nephe-controller`. Below sections sets up access to public cloud account,
+`Nephe Controller`. Below sections sets up access to public cloud account,
```

Comment:
somewhere you use nethe-contoller and somwhere Nephe Controller? Did anyone provide any recommendation when to use what?

Reply:
nephe-controller - > This can be used whenever we are referring to nephe controller binary.
Nephe Controller -> This can be used to refer controller of nephe.

Whenever we are referring to binary, it has to be written as `nephe controller`, while the other can be in plain text Nephe Controller with camel case. This I got from Jianjun's recommendation for antrea-agent.